modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetrics.testcase

```diff
@@ -1013,6 +1013,25 @@ definition {
 			taskName = "Completed");
 	}
 
+	@description = "LRQA-53649 Automate LPS-87560: Verify metrics available or not for CE and DXP customers"
+	@priority = "5"
+	test MetricsAvailableForDXP {
+		property portal.acceptance = "true";
+
+		var key_workflowProcessName = "Single Approver";
+
+		Navigator.openURL();
+
+		GlobalMenu.gotoPortlet(
+			category = "Workflow",
+			panel = "Applications",
+			portlet = "Submissions");
+
+		AssertTextEquals(
+			locator1 = "WorkflowMetrics#METRICS_WORKFLOW_PROCESS_NAME",
+			value1 = "Single Approver");
```

Comment:
Instead of this code, use the same code from `test MetricsUnavailableForCECustomers` but instead of using the `AssertElementNotPresent` at the end, use the `AssertElementPresent`.